### ISSUE TITLE
On the package page, the section below the tab-bar should have the 'main' aria role

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7414]
![image](https://github.com/dart-lang/pub-dev/assets/3865266/1b7909bf-6d7d-4740-8cfc-578ea15997e5)
